src/libponyc/expr/lambda.c

```diff
@@ -39,6 +40,11 @@ static ast_t* make_capture_field(pass_opt_t* opt, ast_t* capture)
       return NULL;
     }
 
+    // lambda captures used before their declaration with their type
+    // not defined are not legal
+    if( !def_before_use(opt, def, capture, name) )
```

Comment:
Can you remove the extra whitespace around the condition expression here, to match the style of the rest of the codebase?